### PR TITLE
Add hosted badge to MetaCPAN

### PR DESCRIPTION
**Pull request by thibaultduponchelle**
metacpanbadges

### CODE REVIEW

**Comment by haarg** [comment]
I'm not sure I understand the purpose of this if it's just showing a static image.

**Comment by thibaultduponchelle** [comment]
> I'm not sure I understand the purpose of this if it's just showing a static image.

Ideally it should include the version but it is not implemented in this pull request.

File: lib/MetaCPAN/Web/Controller/Badge.pm [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package MetaCPAN::Web::Controller::Badge;
+
+use Moose;
+
+BEGIN { extends 'MetaCPAN::Web::Controller' }
+
+use DateTime;
```
Comment (oalders) [inline lib/MetaCPAN/Web/Controller/Badge.pm:7]: This doesn't appear to be used?

**Comment by oalders** [comment]
Thanks for this, @thibaultduponchelle!

> Ideally it should include the version but it is not implemented in this pull request.

Did you want to implement it in a followup or were you hoping someone else would do that?

**Comment by mohawk2** [comment]
There are two failing tests: one has been fixed already so you need to rebase past that, and the other needs you to run the "tidyall".

**Comment by thibaultduponchelle** [comment]
> Did you want to implement it in a followup or were you hoping someone else would do that?

The answer was "I have no idea" :)

Sorry for the delayed answer, I was in vacation, if you don't mind, I will close this *incomplete* PR for now.

**Pull request closed without merging**